packages/netlify-cms-widget-markdown/src/MarkdownControl/Toolbar.js

```diff
@@ -86,9 +87,9 @@ export default class Toolbar extends React.Component {
     t: PropTypes.func.isRequired,
   };
 
-  isHidden = button => {
+  isVisible = button => {
     const { buttons } = this.props;
-    return List.isList(buttons) ? !buttons.includes(button) : false;
+    return !List.isList(buttons) || buttons.includes(button);
```

Comment:
This switch from `isHidden` to `isVisible` is unrelated cleanup, toolbar buttons were handling their own visibility.